multichain-testing/test/send-anywhere.test.ts

```diff
@@ -0,0 +1,142 @@
+    if (!config) return t.fail(`Unknown chain: ${chainName}`);
+
+    const {
+      wallets,
+      provisionSmartWallet,
+      vstorageClient,
+      retryUntilCondition,
+      useChain,
+    } = t.context;
+
+    t.log('Create a receiving wallet for the sendAnywhere transfer');
+    const chain = useChain(chainName).chain;
+
+    t.log('Create an agoric smart wallet to initiate sendAnywhere transfer');
+    const agoricAddr = wallets[`${chainName}${acctIdx}`];
+    const wdUser1 = await provisionSmartWallet(agoricAddr, {
+      BLD: 100_000n,
+      IST: 100_000n,
+    });
+    t.log(`provisioning agoric smart wallet for ${agoricAddr}`);
+
+    const doOffer = makeDoOffer(wdUser1);
+
+    const brands = await vstorageClient.queryData(
+      'published.agoricNames.brand',
+    );
+    const istBrand = Object.fromEntries(brands).IST;
+
+    const apiUrl = await useChain(chainName).getRestEndpoint();
+    const queryClient = makeQueryClient(apiUrl);
+    t.log(`Made ${chainName} query client`);
+
+    const doSendAnywhere = async (amount: Amount) => {
+      t.log(`Sending ${amount.value} ${amount.brand}.`);
+      const wallet = await createWallet(chain.bech32_prefix);
+      const receiver = {
+        chainId: chain.chain_id,
+        value: (await wallet.getAccounts())[0].address,
+        encoding: 'bech32',
+      };
+      t.log('Will send payment to:', receiver);
+      t.log(`${chainName}  offer`);
+      const offerId = `${chainName}-makeSendInvitation-${Date.now()}`;
+      await doOffer({
+        id: offerId,
+        invitationSpec: {
+          source: 'agoricContract',
+          instancePath: [contractName],
+          callPipe: [['makeSendInvitation']],
+        },
+        offerArgs: { destAddr: receiver.value, chainName },
+        proposal: { give: { Send: amount } },
+      });
+
+      // TODO confirm offerResult = undefined
+
+      const { balances } = await retryUntilCondition(
+        () => queryClient.queryBalances(receiver.value),
+        ({ balances }) => 'amount' in balances[0],
+        `${receiver.value} ${amount.value} balance available from sendAnywhere`,
+      );
+
+      t.log(`${receiver.value} Balances`, balances);
+      t.like(balances, [
+        {
+          // XXX consider verifying uist hash
+          amount: String(amount.value),
+        },
+      ]);
+    };
+
+    // between 10n and 100n
+    const makeRandomValue = (min: number, max: number) =>
+      BigInt(Math.floor(Math.random() * (max - min + 1)) + min);
+    // send 3 offers from each account
+    const scenarios = [
+      makeRandomValue(1, 33),
+      makeRandomValue(34, 66),
+      makeRandomValue(67, 100),
+    ];
+    console.log('Scenarios', scenarios);
+
+    for (const scenario of scenarios) {
+      await doSendAnywhere(AmountMath.make(istBrand, scenario));
+    }
+  },
+});
+
+for (const [chainName, accountIdx] of scenarios) {
+  test.serial(sendAnywhereScenario, chainName, accountIdx);
+}
```

Comment:
this is more flexible but since it's a test sometimes it's better to be explicit:
```suggestion
test.serial(sendAnywhereScenario, 'osmosis', 1);
test.serial(sendAnywhereScenario, 'cosmoshub', 2);
```

Though I'd also suggest dropping the index number and having a lookup for the test account for each named chain.